lib/swagger/service/index.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-
+const namespace = require('continuation-local-storage').getNamespace('requestNamespace');
```

Comment:
split into two lines:

```
const cls       = ...
const namespace = ...
```

remove `;` — we don't use them anywhere, as they are useless.